### PR TITLE
add rack metrics

### PR DESCRIPTION
**Pull request by StevenLuMT**
Master Issue: no-issue

### Motivation

- pulsar lacks the bookie rack's metric in use

### Modifications

- Add main class BookkeeperIsolationGroupsStats,two categories of metrics:

1. current rack group's rack metrics
2. all rack group's writable rack metrics

- metric main class is inited  in class PulsarService
- add equals method in class RawBookieInfo
- add comments in class BookiesRackConfiguration

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): ( no)
  - The public API: (no)
  - The schema: ( no )
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation
  
- [x] `no-need-doc`

### CODE REVIEW

File: pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/RawBookieInfo.java [hunk 1]
```diff
@@ -23,6 +23,8 @@
 import lombok.NoArgsConstructor;
 import lombok.ToString;
 
+import java.util.Objects;
```
Comment (AnonHxy) [inline pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/RawBookieInfo.java:26]: checkstyle failed.  Should reorder tht imports. Maybe like this：
```
import java.util.Objects;
import lombok.NoArgsConstructor;
import lombok.ToString;
```

**Comment by StevenLuMT** [comment]
@Jason918  can you approval this workflows?

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java [hunk 2]
```diff
@@ -810,6 +811,9 @@ public void start() throws PulsarServerException {
                     config.getClusterName(), config);
 
             state = State.Started;
+
+            new BookkeeperIsolationGroupsStats(pulsarResources, getBookKeeperClient(), config,
```
Comment (Jason918) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java:815]: A static method seems better?

**Comment by tisonkun** [comment]
Closed as stale and conflict. Please rebase and resubmit the patch if it's still relevant.

I think this patch is good to go. So @StevenLuMT you can always revive this thread.

**Pull request closed without merging**